Web/CaseStudyAssignment-4/CaseStudyAssignment-4/WebContent/editCar.jsp

```diff
@@ -0,0 +1,138 @@
+<%@page import="com.metacube.carportal.dbconfig.ConnectionFactory"%>
+<%@page import="com.metacube.carportal.db.helper.CarPortalDao"%>
+<%@page import="com.metacube.carportal.model.Car"%>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<%@page import="java.util.List"%>
+<%@page import="java.sql.Connection"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
```

Comment:
ACTION: Do we need to create new page for edit car? can we create same page for create and edit car?